content/en/docs/setup/install/multicluster/index.md

```diff
@@ -0,0 +1,744 @@
+            port: 15443
+        NETWORK2:
+          endpoints:
+          - fromRegistry: CLUSTER2
+          gateways:
+          - registryServiceName: istio-eastwestgateway.istio-system.svc.cluster.local
+            port: 15443
+EOF
+$ istioctl install --context=${CTX_CLUSTER2} -f cluster2.yaml
+{{< /text >}}
+
+As we did with `CLUSTER1` above, install a gateway in `CLUSTER2` that is dedicated
+to east-west traffic and expose user services.
+
+{{< text bash >}}
+$ CLUSTER=CLUSTER2 NETWORK=NETWORK2 \
+    samples/multicluster/gen-eastwest-gateway.sh | \
+    kubectl apply --context=${CTX_CLUSTER2} -f -
+{{< /text >}}
+
+{{< text bash >}}
+$ kubectl --context=${CTX_CLUSTER2} apply -n istio-system -f \
+    samples/multicluster/expose-services.yaml
+{{< /text >}}
+
+- **Enable Endpoint Discovery for `CLUSTER1` and `CLUSTER2`**
+
+Install a remote secret in `CLUSTER2` that provides access to `CLUSTER1`’s API server.
+
+{{< text bash >}}
+$ istioctl x create-remote-secret \
+  --context=${CTX_CLUSTER1} \
+  --name=CLUSTER1 | \
+  kubectl apply -f - --context=${CTX_CLUSTER2}
+{{< /text >}}
+
+Install a remote secret in `CLUSTER1` that provides access to `CLUSTER2`’s API server.
+
+{{< text bash >}}
+$ istioctl x create-remote-secret \
+  --context=${CTX_CLUSTER2} \
+  --name=CLUSTER2 | \
+  kubectl apply -f - --context=${CTX_CLUSTER1}
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="Primary-Remote" category-value="primary-remote" >}}
+
+The following steps will install the Istio control plane on `CLUSTER1` (the
+{{< gloss >}}primary cluster{{< /gloss >}}) and configure `CLUSTER2` (the
+{{< gloss >}}remote cluster{{< /gloss >}}) to use the control plane in `CLUSTER1`.
+Both clusters reside on the `NETWORK1` network, meaning there is direct
+connectivity between the pods in both clusters.
+
+`CLUSTER1` will be configured to observe the API Servers in both clusters for
+endpoints. In this way, the control plane will be able to provide service
+discovery for workloads in both clusters.
+
+Service workloads communicate directly (pod-to-pod) across cluster boundaries.
+
+Services in `CLUSTER2` will reach the control plane in `CLUSTER1` via a
+dedicated gateway for [east-west](https://en.wikipedia.org/wiki/East-west_traffic)
+traffic.
+
+{{< image width="75%"
+    link="primary-remote.svg"
+    caption="Primary and remote clusters on the same network"
+    >}}
+
+- **Configure `CLUSTER1` as a primary with control plane exposed**
+
+Install Istio on `CLUSTER1`:
+
+{{< text bash >}}
+$ cat <<EOF > ./cluster1.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: MESH1
+      multiCluster:
+        clusterName: CLUSTER1
+      network: NETWORK1
+      meshNetworks:
+        ${NETWORK1}:
+          endpoints:
+          - fromRegistry: CLUSTER1
+          - fromRegistry: CLUSTER2
+          gateways:
+          - registryServiceName: istio-eastwestgateway.istio-system.svc.cluster.local
```

Comment:
Similar comment as earlier - Is gateway required here?

Reply:
Ok i think it is required here for workloads on remote cluster to reach out to istiod... but not for workloads from remote cluster to reach out to cluster 1.

Reply:
yup, exactly.